upstream-community-operators/kubefed-operator/kubefed-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -281,71 +285,129 @@ spec:
     quickly get up and running with this powerful concept. Federation is a key
     part of any Hybrid Cloud capability.
 
-    **Important Note**: This operator is purely for prototyping purposes and not
-    supported by any upstream community at this time.
-
-
-    **Important Note**: Currently, while in preview, this operator does not
-    support automatic upgrades. You must remove the old version of the operator
+    **Support**: 
+    
+    The KubeFed operator is purely for prototyping purposes and not
+    supported by any upstream community at this time. Currently, 
+    while in preview, this operator does not support automatic 
+    upgrades. You must remove the old version of the operator
```

Comment:
`Note that at the moment uninstalling the operator doesn't delete the custom resources it reconciles. Users may optionally want to delete the custom resources and the associated CRDs before reinstalling the upgraded version of the operator`

Reply:
Fixed